wavedroplet.js

```diff
@@ -116,15 +124,18 @@ d3.json('/json/' + get_query_param('key')[0], function(error, json) {
 
 function get_query_param(param) {
     var urlKeyValuePairs = {}
-    window.location.search.split("?")[1].split("&").forEach(function(d) {
-        var m = d.split("=");
+    window.location.hash.split("#")[1].split("&").forEach(function(d) {
+        var m = d.split("~");
         urlKeyValuePairs[m[0]] = m[1]
     })
+    console.log(window.location.hash, urlKeyValuePairs)
     return urlKeyValuePairs[param].split(',')
 }
 
-function to_stream_key(d) {
-    return d['ta'].replace(/:/g, '') + '_' + d['ra'].replace(/:/g, '');
+function to_stream_key(d, aliases) {
+    var from = aliases[d['ta']] || d['ta']
+    var to = aliases[d['ra']] || d['ra']
+    return from.replace(/:/g, '') + '_' + to.replace(/:/g, '');
```

Comment:
I'd rather use something instead of _ here, to better indicate the direction of the stream.  A>B, A->B, etc.